### ISSUE TITLE
0.75l of blueberry flavored vodka is enough to kill a man (don't ask why I know)

### ISSUE THREAD

**Issue by Karol1223** [CleverRaven/Cataclysm-DDA#73888]
### Describe the bug

0.75l of boozeberry is a lethal dose (sometimes, read below) and gets you Dead Drunk
0.75l of vodka only gets you Wasted

### Attach save file

N/A

### Steps to reproduce

First things first - it's not 100% reproducability rate. I couldn't get to reproduce it myself. @TheMurderUnicorn helped me and they were able to reproduce it a few times, but I only had it happen to me once

Spawning 3 items (18 charges) of boozeberry and drinking it at once *sometimes* kills a character

### Expected behavior

Admittedly, I am not aware how much vodka is enough to kill you. Google gives results anywhere between 1l and 1.75l, but boozeberry is more diluted yet. You use 140ml of vodka to make 250ml of boozeberry, so it shouldn't get you *more* drunk than vodka.

### Screenshots

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/68503002/c9649f56-3321-47ab-b19a-da9f83e304f8)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.22631.3593 (23H2)
- Game Version: cdda-experimental-2024-05-08-1351 f2d1760 [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth],
    Project Pangaea [ProjectPangaea],
    Stats Through Kills [stats_through_kills]
]

**Comment by TheMurderUnicorn** [comment]
Yeah I don't have much to add, I was able to reproduce it sometimes.  A lot of times I'd pass out before drinking it all, which would prevent me from getting all portions and thus stop me from dying.  But it's murdering me for sure, here's a pic of the log after dying. 
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/41651793/67faefea-ad06-4746-83aa-cebc843e940c)
